README.md

```diff
@@ -1,41 +1,43 @@
+Patcher.xyz is a platform designed to simplify the process of transcribing and sharing patch notes. It allows users to build patches by clicking the INs and OUTs of modules in the "Patch Builder", visualize the signal flow, and share or save the patches with ease. It also features a rack builder for sorting modules.
 
-Patches are built by clicking the INs and OUTs of modules added to your collection in the "Patch Builder". You can then see the signal flow in our unique graph visualizer.
+## Usage
 
-Similarly to modulargrid, we also have a rack builder that can sort modules while dragging and dropping.
+### What can I do with the patch builder?
 
-## What can I do?
+The patch builder is a versatile tool that allows you to connect CV inputs and CV outputs quickly and easily. For instance, you can create a complex signal flow by connecting multiple modules, visualize it in our unique graph visualizer, and then share the URL or save the list of connections as text. All patches are saved on the website and can be shared with a simple link.
 
-The patch builder allows connecting CV inputs and CV outputs in an easy and fast way.
+### Do I need a list of all modules and their information to use the patch builder?
 
-You can then share the URL or save the list of connections as text. Patches are saved on the website itself and can be shared with a simple link.
+Yes, but don't worry! We already have a substantial database of inputs and outputs for each module, thanks to the contributions of our community. We're always looking for more help to map all modules' CVs.
 
-### But doesn't this require having a list of all modules and their information?
+## Contribution
 
-Yes, however, we are not starting from scratch!
+### How can I contribute to the project?
 
-Since one of the objectives is to maintain a database of inputs and outputs of each module, we ask for the help of the community to map all modules' CVs. We already have a lot of data mapped and ready.
+We're always open to feedback and suggestions for improvement. The best way to contribute is to join our discord channel, which is our main hub for communication. You can also help us by mapping the CVs of modules.
 
-## Great! How can I help?
+### How can I report issues?
 
-The first public announcements of this idea are being made and initial feedback is being collected. Please let us know your thoughts on this and what you would like to see improved.
+If you encounter any issues while using patcher.xyz, please report them on our Github issue tracker. We appreciate your help in making our platform better.
 
-The best way to stay informed about development is to subscribe to the discord channel, which is the central communication hub.
+### How can I request features?
 
-## Ok that's nice, where can I try it?
+Feature requests are welcome! You can submit your ideas on our Github issue tracker or discuss them in our discord channel.
```

Comment:
do not send the users to the issue tracker. Instead redirect them to discord.

Reply:
change the content of the text. do not send the users to the issue tracker. Instead redirect them to discord.do not send the users to the issue tracker. Instead redirect them to discord.

Reply:
change this line to: `Feature requests are welcome! You can submit your ideas and discuss them in our discord channel.`

Reply:
change this line to:

Feature requests are welcome! You can submit your ideas and discuss them in our discord channel.